Cargo.toml

```diff
@@ -33,6 +35,8 @@ typed-builder = { version = "0.20.0", optional = true }
 default = ["aes-session"]
 serde = ["dep:serde", "dep:serde_bytes"]
 builder = ["typed-builder"]
+"embedded-hal-v0.2.7" = ["dep:embedded-hal-v0_2_7"]
+"embedded-hal-v1.0" = ["dep:embedded-hal-v1_0"]
```

Comment:
```suggestion
"embedded-hal-v0.2.7" = ["embedded-hal-v0_2_7"]
"embedded-hal-v1.0" = ["embedded-hal-v1_0"]
```

Reply:
That does not compile for me:

feature `embedded-hal-v0.2.7` includes `embedded-hal-v0_2_7`, but `embedded-hal-v0_2_7` is an optional dependency without an implicit feature
Use `dep:embedded-hal-v0_2_7` to enable the dependency.